packages/suite/src/components/suite/NavigationBar/components/NavigationActions/components/ActionItem/index.tsx

```diff
@@ -78,8 +78,8 @@ const Indicator = styled.div`
     width: 10px;
     border-radius: 50%;
     position: absolute;
-    top: 10px;
-    right: 10px;
+    top: 0px;
+    right: -1px;
```

Comment:
Looks like that this is not shared with Bell

<img width="248" alt="Screenshot 2021-09-09 at 10 32 03" src="https://user-images.githubusercontent.com/33235762/132651923-9966a39a-3836-4687-b431-e3b0d84c8c9d.png">

Reply:
73a67f5